domain/src/main/java/org/oppia/android/domain/platformparameter/BUILD.bazel

```
@@ -105,6 +78,15 @@ kt_android_library(
     ],
 )
 
+kt_android_library(
+    name = "debug_impl",
+    srcs = ["PlatformParameterControllerDebugImpl.kt"],
+    visibility = ["//:oppia_debug_visibility"],
+    deps = [
+        ":prod_impl",
+    ],
```

Formatting suggestion -- since it is a single dependency?

```suggestion
    deps = [":prod_impl"],
```

---

updated